components/TokenTable/index.js

```diff
@@ -174,11 +176,11 @@ const TokenTable = ({ tokens, vetPrice, itemMax = 10, useTracked = false }) => {
                     setSortDirection(sortedColumn !== SORT_FIELD.NAME ? true : !sortDirection)
                   }}
                 >
-                  {ifBelow680 ? 'Symbol' : 'Name'} {sortedColumn === SORT_FIELD.NAME ? (!sortDirection ? '↑' : '↓') : ''}
+                  {'Pair'} {sortedColumn === SORT_FIELD.NAME ? (!sortDirection ? '↑' : '↓') : ''}
                 </ClickableText>
               </Flex>
             </th>
-            {!ifBelow640 && (
```

Comment:
I think we can remove the name and symbol and do something like VEX-VET, SHA-VET, etc. Similar to https://v2.info.uniswap.org/pairs